packages/@webex/plugin-meetings/test/unit/spec/reconnection-manager/index.js

```diff
@@ -2,37 +2,38 @@ import 'jsdom-global/register';
 import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import sinon from 'sinon';
+import {Defer} from '@webex/common';
 import ReconnectionManager from '@webex/plugin-meetings/src/reconnection-manager';
+import ReconnectionNotStartedError from '@webex/plugin-meetings/src/common/errors/reconnection-not-started';
+import TriggerProxy from '@webex/plugin-meetings/src/common/events/trigger-proxy';
+import Metrics from '@webex/plugin-meetings/src/metrics';
 import { RECONNECTION } from '../../../../src/constants';
 import LoggerProxy from '../../../../src/common/logs/logger-proxy';
 import LoggerConfig from '../../../../src/common/logs/logger-config';
-
 const {assert} = chai;
 
 chai.use(chaiAsPromised);
 sinon.assert.expose(chai.assert, {prefix: ''});
 
 describe('plugin-meetings', () => {
   describe('ReconnectionManager.reconnect', () => {
-    const sandbox = sinon.createSandbox();
```

Comment:
I've removed the sandbox, because it's not needed. I think this is something that was required in very old versions of sinon, now sinon has a default sandbox always there.